test_ir.py

```diff
@@ -18,5 +18,14 @@ class IRkernelTests(jkt.KernelTests):
     complete_code_samples = ['1', "print('hello, world')", "f <- function(x) {\n  x*2\n}"]
     incomplete_code_samples = ["print('hello", "f <- function(x) {\n  x*2"]
 
+class InstallspecTests(jkt.KernelTests):
+    # just a small test, it's the same kernel after all...
+    kernel_name = "testir"
+
+    language_name = "R"
+
+    code_hello_world = "print('hello, world')"
+
+
```

Comment:
I wrote it, so that's expected. I'm completely inconsistent with quotes in code. The only time I pay attention to them is when I want to put one or other kind of quote inside the string itself. I also don't particularly care about making them consistent, so don't feel we need to 'clean up' the Python code.